src/DynamoCore/Engine/NodeToCode/NodeToCode.cs

```diff
@@ -1210,6 +1210,26 @@ public static void ReplaceWithShortestQualifiedName(ClassTable classTable, IEnum
             }
             #endregion
 
+            #region Step 5 Escape StringInputNodes
+            foreach ((NodeModel nodeModel, IEnumerable<AssociativeNode> astNodes) in allAstNodes)
```

Comment:
Seems like we are doing this exact loop a bunch of times in these 5 steps. Would it be possible to add our code to one of the existing loops?